pom.xml

```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>ludii</groupId>
+    <artifactId>parent</artifactId>
+    <version>1.3.1</version>
+    <packaging>pom</packaging>
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <ludii.version>1.3.1</ludii.version>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <commons-rng.version>1.4</commons-rng.version>
+    </properties>
+    <modules>
+        <module>AI</module>
+        <module>Common</module>
+        <module>Core</module>
+        <module>Evaluation</module>
+        <module>Features</module>
+        <module>Generation</module>
+        <module>Language</module>
+        <module>LudiiDocGen</module>
+        <module>Manager</module>
+        <module>Mining</module>
+        <module>Player</module>
+        <module>PlayerDesktop</module>
+        <module>ViewController</module>
+    </modules>
+    <dependencyManagement>
+        <dependencies>
```

Comment:
Please review these dependencies. I tried to get as close as possible as the ones already in use.